deploy-cluster-aws/awsdeploy.sh

```diff
@@ -116,7 +117,7 @@ fab upgrade_setuptools
 
 if [ "$WHAT_TO_DEPLOY" == "servers" ]; then
     # (Re)create the RethinkDB configuration file conf/rethinkdb.conf
-    python create_rethinkdb_conf.py
+    python create_rethinkdb_conf.py --bind-http-to-localhost $BIND_HTTP_TO_LOCALHOST
```

Comment:
As I said, it's basically a matter of taste. Moreover, it's not broken, so I'm reluctant to "fix" it.